scipy/stats/_sampling.py

```diff
@@ -1140,15 +1140,11 @@ class RatioUniforms:
         The upper bound of the bounding rectangle in the v-direction.
     c : float, optional.
         Shift parameter of ratio-of-uniforms method, see Notes. Default is 0.
-    random_state : {None, int, `numpy.random.Generator`,
```

Comment:
It looks to me like `random_state` cannot just be deleted from the docs; it should be mentioned as an alias, since it's not removed and is widely used.